pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/BrokerOperabilityMetrics.java

```diff
@@ -34,6 +35,11 @@
     private final DimensionStats zkWriteLatencyStats;
     private final DimensionStats zkReadLatencyStats;
     private final String brokerName;
+    private final AtomicLong connectionTotalCreatedCount;
```

Comment:
Prefer `LongAdder` instead of `AtomicLong` since it will have less contention issues.

Reply:
OK  i will change the atomicLong to LongAdder.